concepts/closures/links.json

```diff
@@ -11,10 +11,6 @@
     "url": "https://en.wikipedia.org/wiki/Closure_(computer_programming)",
     "description": "wiki-closure"
   },
-  {
-    "url": "https://whatthefork.is/closure",
```

Comment:
It was such a good article.

How about we link to https://web.archive.org/web/20210702195218if_/https://whatthefork.is/closure instead?

Reply:
Sure, if it's that good 😁